schema/method_units/unit_base_physical.json

```diff
@@ -0,0 +1,41 @@
+{
+    "schemaId": "method-unit-base-physical",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "physical unit method schema (base)",
+    "allOf": [
+        {
+            "$ref": "unit_base.json"
+        },
+        {
+            "$ref": "#/definitions/relations-to-models-and-mathematical-methods"
+        }
+    ],
+    "definitions": {
+        "relations-to-models-and-mathematical-methods": {
+            "properties": {
+                "mathematicalMethodsCategoryPaths": {
+                    "description": "Tracks from which mathematical unit methods this physical method derives",
+                    "type": "array",
+                    "items": {
+                        "items": {
+                            "type": [
+                                "string"
+                            ]
+                        }
+                    }
+                },
+                "modelCategoryPaths": {
```

Comment:
1. Let's add a path item to reusable and put a note to come back to define it through regex when newer version of JSON schema is adopted
2. Let's define and array of paths and use here